widgets/heatmap/options.go

```diff
@@ -29,21 +28,25 @@ type Option interface {
 
 // options stores the provided options.
 type options struct {
-	// The default value is 3
 	cellWidth      int
+	hideXLabels    bool
+	hideYLabels    bool
 	xLabelCellOpts []cell.Option
 	yLabelCellOpts []cell.Option
 }
 
 // validate validates the provided options.
 func (o *options) validate() error {
-	return errors.New("not implemented")
+	//if got, min := o.cellWidth, DefaultCellWidth; got < min {
```

Comment:
This block might be left commented out by accident?

Applies to lines 49 and 65 below too:
```go
opt := &options{
    //cellWidth: DefaultCellWidth,
}

//const DefaultCellWidth = 3
```